CHANGELOG.md

```diff
@@ -7,8 +7,8 @@ This file is used to list changes made in each version of the AWS ParallelCluste
 ------
 **CHANGES**
 - Upgrade Slurm to 23.02.5 (from 23.02.4).
-- Upgrade Pmix to 4.2.6 (from 3.2.3).
-- Upgrade libjwt to 1.15.3 (from 1.12.0).
+  - Upgrade Pmix to 4.2.6 (from 3.2.3).
```

Comment:
Extra space?

Reply:
I cannot amend this PR, so I had to recreat a PR https://github.com/aws/aws-parallelcluster-cookbook/pull/2459